gobcore/logging/logger.py

```diff
@@ -216,14 +221,17 @@ def _log(self, level, msg, kwargs=None):
         :param kwargs:
         :return: None
         """
-        logger = getattr(Logger._logger[self._name], level)
+        level_logger = getattr(self.log_instance, level)
         assert logger, f"Error: invalid logging level specified ({level})"
```

Comment:
assert level_logger here (if this didnt fail it is also not properly tested)

Reply:
fixed